src/lib.rs

```diff
@@ -43,13 +43,16 @@
 #![cfg_attr(feature="clippy", feature(plugin))]
 #![cfg_attr(feature="clippy", plugin(clippy))]
 
+#![allow(dead_code)]
+#![allow(unused_variables)]
```

Comment:
Yep, mentioned that in the last line in first pr post
Btw dead code means here unused functions.
So yes, this omits lots of warnings and its necessity probably should be discussed before merge